apps/web-giddh/src/app/settings/settings.component.ts

```diff
@@ -178,6 +178,10 @@ export class SettingsComponent implements OnInit, OnDestroy {
         switch (childTab) {
             case 'payment':
                 return 4;
+            case 'ecommerce':
```

Comment:
This belongs to my task, This shouldn't go with your task

Reply:
I asked to you which one, you told me incoming